src/main/java/com/basho/riak/client/core/converters/TimeSeriesPBConverter.java

```diff
@@ -21,28 +18,28 @@
 {
     private TimeSeriesPBConverter() {}
```

Comment:
IMO better to introduce complete Abstract Factory pattern in case we decide to move forward with both mutable and immutable APIs